bazel/pipeline.bzl

```diff
@@ -40,65 +40,82 @@ def sematic_pipeline(
         registry: URI of the container registry to use to register
             the container image
 
-        repository: (optional) container repository for the image
+        repository: container repository for the image
 
         data: (optional) data files to add to the image
 
         base: (optional) label of the base image to use.
 
+        bases: (optional)
```

Comment:
This argument is not meant to be used in conjunction with `base`, but instead of it. Also, we don't intend to document this option as we don't want to encourage users to use it.
I'm happy to rename it to `base_mapping` if you prefer.

Reply:
The issue here is that `base` has a default value.
So if users only pass `bases`, then both `base` and `bases` will be non-null. So the check you suggest would fail. Users would then need to explicitly set `base` to `None`, which is impractical.
The behavior currently implemented is the best behavior considering the constraints.
Also, `bases` will not be documented and only used with user explicitly requiring this.